chain/messagepool/check.go

```diff
@@ -32,7 +32,10 @@ func (mp *MessagePool) CheckMessages(ctx context.Context, protos []*api.MessageP
 // CheckPendingMessages performs a set of logical sets for all messages pending from a given actor
 func (mp *MessagePool) CheckPendingMessages(ctx context.Context, from address.Address) ([][]api.MessageCheckStatus, error) {
 	var msgs []*types.Message
-	mp.lk.RLock()
```

Comment:
Can we remove the `lk` lock from the MessagePool struct, or is it still used somewhere?

Reply:
Yes!! We can! Deleted lk in b0f3311d8!